timesketch/frontend-ng/src/components/UploadForm.vue

```diff
@@ -145,17 +146,8 @@ limitations under the License.
         </v-container>
         <v-card-actions>
           <v-spacer></v-spacer>
-          <v-btn
-            color="primary"
-            text
-            @click="
-              clearFormData()
-              dialog = false
-            "
-          >
-            Cancel
-          </v-btn>
-          <v-btn v-if="fileName" color="primary" text @click="clearFormData()"> Select another file </v-btn>
+          <v-btn text @click="dialog = false"> Cancel </v-btn>
+          <v-btn v-if="fileName" text @click="clearFormData()"> Select another file </v-btn>
           <v-btn color="primary" text @click="submitForm()" v-if="!(error.length > 0 || !fileName)"> Submit </v-btn>
```

Comment:
This form cannot be submitted via hitting the enter key. Might be worth adding a `@keyup.enter=submitForm()` event for this dialog?

Reply:
This form is a bit too complex for that and we need better form validation before we can add that. This will happen in a follow PR